setup.py

```diff
@@ -34,8 +34,10 @@ def get_all_extras() -> Dict:
 
     ethereum_ledger_deps = [
         "web3==5.12.0",
-        "ipfshttpclient==0.6.1",
```

Comment:
we need to keep this here; it can otherwise lead to issues with our scripts

Reply:
I thought that scripts mostly related to development

Reply:
Please revert (see slack)